### ISSUE TITLE
"Presets & settings" button in effect dialogs does not have an access key

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#3489]
### Steps to reproduce

Open an effect dialog

### Expected behavior

Presets & settings button has an access key

### Actual behavior

Presets & settings button does not have an access key

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

### Additional context

The Presets & settings button replaces the Manage button in previous versions of Audacity.
The Manage button had the access key M. So once an access key is assigned (which won't be M!), then all the built-in effects will need to be checked to see if that access key is used for another control, and if so access key or keys reassigned so that access keys remain unique in a dialog.

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
